crates/sui-types/src/committee.rs

```diff
@@ -28,12 +26,26 @@ pub type StakeUnit = u64;
 
 pub type CommitteeDigest = [u8; 32];
 
+// The voting power, quorum threshold and max voting power are defined in the `voting_power.move` module.
+// We're following the very same convention in the validator binaries.
+
+/// Set total_voting_power as 10_000 by convention. Individual voting powers can be interpreted
+/// as easily understandable basis points (e.g., voting_power: 100 = 1%, voting_power: 1 = 0.01%).
+/// Fixing the total voting power allows clients to hardcode the quorum threshold and total_voting power rather
+/// than recomputing these.
+pub const TOTAL_VOTING_POWER: StakeUnit = 10_000;
```

Comment:
Since these are not actually stake units anymore should we use a different type?

Reply:
We should. But let's refactor that after the cut.. I am still fighting with some strange simtest failure with this PR